receiver/prometheusremotewritereceiver/changelog/extensions-prometheusremotewritereceiver-countermetrics.yaml

```diff
@@ -0,0 +1,3 @@
+change_type: enhancement
+component: receiver/prometheusremotewritereceiver
+note: Add proper counter metric handling with monotonic and cumulative settings
```

Comment:
Nice, thanks for caring about changelog! I think you misplaced it though, changelog files are stored in the `.chloggen` directory